docs/tekton-controller-performance-configuration.md

```diff
@@ -49,6 +49,8 @@ spec:
 
 Now, the ThreadsPerController, QPS and Burst have been changed to be `32`, `50` and `50`.
 
+You can also set `THREADS_PER_CONTROLLER`, `KUBE_API_QPS` and `KUBE_API_BURST` environment variables to overwrite default values.
```

Comment:
+1

Thanks for updating the docs. Perhaps you could say (even if it's already implied), that command line args will take precedence.